### ISSUE TITLE
Change group Site settings headings

### ISSUE THREAD

**Issue by boonebgorges** [cuny-academic-commons/openlab-theme#229]
From @jennaspevack 

https://www.dropbox.com/sh/i9krg6tdk1mzebq/AACnLoLjyoYmKxFfeP2rzupPa/Settings%3ESite.png?dl=0

This came from the agreed-upon designs created by @samradd. Can I get some clarification?

**Comment by jennaspevack** [comment]
I find it a bit confusing, because it's not mentioned anywhere else on the page - the section says just says Site (and it's a bit wordy). Not a big deal though. Fine to reject this ticket.  :)

**Comment by boonebgorges** [comment]
> Fine to reject this ticket. :)

Done :)
